nav2_amcl/src/amcl_node.cpp

```diff
@@ -1338,3 +1053,146 @@ AmclNode::createMotionModel()
     motionModel_ = new DifferentialMotionModel(alpha1_, alpha2_, alpha3_, alpha4_);
   }
 }
+
+void
+AmclNode::initAmclParams()
+{
+  // Grab params off the param server
+  get_parameter_or_set("use_map_topic_", use_map_topic_, false);
+  get_parameter_or_set("first_map_only_", first_map_only_, false);
+  double save_pose_rate;
+  get_parameter_or_set("save_pose_rate", save_pose_rate, 0.5);
+  save_pose_period = tf2::durationFromSec(1.0 / save_pose_rate);
+  get_parameter_or_set("laser_min_range", laser_min_range_, -1.0);
+  get_parameter_or_set("laser_max_range", laser_max_range_, 12.0);
```

Comment:
I'd set this much higher, make the defaults just use full range. Something like 100.0 meters